The command `make install run` doesn't work

When you install the project for the first time, we expect to be able to run `make install run` and get a functionning example.

Instead, I have the following error message:

```
Sending build context to Docker daemon  227.3kB
Step 1/3 : FROM golang:1.8
 ---> 0d283eb41a92
Step 2/3 : RUN mkdir /code
 ---> Using cache
 ---> 287a2947e487
Step 3/3 : WORKDIR /code
 ---> Using cache
 ---> 99fb230c6731
Successfully built 99fb230c6731
Successfully tagged quarto-go:latest
src/quarto/main.go:4:2: cannot find package "quarto/server" in any of:
	/usr/local/go/src/quarto/server (from $GOROOT)
	/go/src/quarto/server (from $GOPATH)
Makefile:21 : la recette pour la cible « run » a échouée
make: *** [run] Erreur 1
```

It might be just a GOPATH issue.

--- a/.gitignore
+++ b/.gitignore
@@ -1,1 +1,2 @@
-*.exe
+pkg/
+src/github.com/

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,14 +1,16 @@
 # https://travis-ci.com/marmelab/quarto-go
 
+sudo: required
 language: go
+
+services:
+    - docker
 
 go: "1.8"
 
-install:
-  - GOPATH=$(pwd) && go get github.com/gorilla/mux
-  - GOPATH=$(pwd) && go get github.com/ahl5esoft/golang-underscore
+install: make install
 
-script: GOPATH=$(pwd) && make test-local
+script: make test
 
 branches:
   only:

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,3 +1,2 @@
 FROM golang:1.8
-RUN mkdir /code
-WORKDIR /code
+WORKDIR /go

--- a/Makefile
+++ b/Makefile
@@ -24,9 +24,6 @@ run: ## Start the game
 test: ## Test the code
 	$(BIN) go test -v ./src/tests
 
-test-local:
-	cd ./src/tests/ && go test -run ''
-
 lint: ## Check the code syntax and rules
 	$(BIN) gofmt -w ./src
 

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,8 @@
 MAKEFLAGS += --silent
 
+UID = $(shell id -u)
+GID = $(shell id -g)
+
 .PHONY: help install run lint
 
 help:
@@ -8,7 +11,9 @@ help:
 BIN = docker run \
 	-it \
 	--rm \
+	--user "${UID}:${GID}" \
 	-v "${PWD}:/go" \
+	-v "/etc/passwd:/etc/passwd:ro" \
 	-p "8080:8080" \
 	--name quarto-go \
 	quarto-go

--- a/Makefile
+++ b/Makefile
@@ -2,13 +2,14 @@ MAKEFLAGS += --silent
 
 .PHONY: help install run lint
 
-help: 
+help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 BIN = docker run \
 	-it \
 	--rm \
-	-v "${PWD}:/code" \
+	-v "${PWD}:/go" \
+	-p "8080:8080" \
 	--name quarto-go \
 	quarto-go
 

